package.json

```diff
@@ -21,15 +21,15 @@
     "mkdirp": "^0.5.6"
   },
   "devDependencies": {
-    "vows": "^0.8.3"
+    "jest": "^24.9.0"
   },
   "main": "./lib/portfinder",
   "types": "./lib/portfinder.d.ts",
   "scripts": {
-    "test": "vows test/*-test.js --spec"
+    "test": "jest --runInBand"
```

Comment:
Jest will try to run the test files in parallel via workers, which breaks the suite given that tests re-use the same ports. Given how quickly it runs (< 1s), I think the lack of parallelization doesn't matter.